src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1342,6 +1448,64 @@ public static async Task<Stream> ReadAsStreamUncompressedAsync(HttpResponseMessa
             return true;
         }
 
+        private bool ShouldBroadcast<TResult>(RavenCommand<TResult> command)
+        {
+            if (command is IBroadcast == false)
+                return false;
+
+            if (TopologyNodes == null || 
+                TopologyNodes.Count < 2)
+                return false;
+
+            return true;
+        }
+
+        private async Task<TResult> Broadcast<TResult>(RavenCommand<TResult> command, SessionInfo sessionInfo, CancellationToken token)
+        {
+            if (command is IBroadcast == false)
```

Comment:
```suggestion
            if (command is IBroadcast broadcast == false)
```

Reply:
funny, compilation failed because "Use of unassigned local variable 'broadcast".
But `!(command is IBroadcast broadcast)` works.
Will use girsha's solution..